ui/modal/info_modal.go

```diff
@@ -212,7 +212,13 @@ func (in *InfoModal) actionButtonsLayout() layout.Widget {
 					}
 
 					in.btnPositve.Text = in.positiveButtonText
-					in.btnPositve.Background, in.btnPositve.Color = in.Theme.Color.Surface, in.Theme.Color.Primary
+
+					bgCol, textCol := in.Theme.Color.Surface, in.Theme.Color.Primary
+					if in.btnPositve.IsDisabled() {
+						bgCol, textCol = in.Theme.Color.Gray, in.Theme.Color.Surface
+					}
+					in.btnPositve.Background, in.btnPositve.Color = bgCol, textCol
```

Comment:
Rather than have this checked and colour set over and over in different parts of the project, it would be better if the colours were set from the button widget when the Disable method is called.